components/benefit_sponsors/app/models/benefit_sponsors/services/new_profile_registration_service.rb

```diff
@@ -204,7 +204,7 @@ def pluck_profile(organization)
 
       def is_benefit_sponsor_already_registered?(user, form)
         if user.person.present? && user.person.has_active_employer_staff_role?
-          form.profile_id = user.person.active_employer_staff_roles.first.benefit_sponsor_employer_profile_id.to_s
+          form.profile_id = user.person.active_employer_staff_roles.where(:benefit_sponsor_employer_profile_id.exists => true).first.benefit_sponsor_employer_profile_id.to_s
           return false
```

Comment:
@ymhari any chance benefit_sponsor_employer_profile_id can fail if the query returns nothing? Where no benefit_sponsor_employer_profile_id exists?